tests/release/e2e-test-default-bundle.go

```diff
@@ -124,7 +124,7 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-738]test-release-service-default-
 			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
 		})
 
-		It("verifies that the PipelineRun in dev namespace succeeded.", func() {
+		It("verifies that the PipelineRun in dev namespace succeeded.", FlakeAttempts(flakeAttemptsTimes), func() {
```

Comment:
@kasemAlem adding `FlakeAttempts` decorator to the test spec results in retrying the content of the test spec. 

In this case it means that if this spec failed, it will try to find a matching pipelinerun again, and waits until it succeeds. But it actually won't succeed, because the PipelineRun has already failed.

So we need to come up with a way to **re-trigger the PipelineRun** in case it failed on [`CouldntGetTask`](https://issues.redhat.com/browse/SRVKP-2749) (which is the issue we're trying to workaround by implementing the retries)

I implemented the solution for **build pipelineruns** (i.e. pipelineruns triggered by creation of the component) [here](https://github.com/redhat-appstudio/e2e-tests/pull/535) and applied that to the test specs that work with build pipelineruns also to "release" tests.

Implementing a solution for other pipelinerun types (i.e. release, integration) will be a slightly different. And regarding the "release" pipeline retries - I think that's a main focus of HACBS-2249